Core/Inc/global_definitions.h

```diff
@@ -12,7 +12,12 @@
 #include "stdint.h"
 
 // cores do LED da ECU
-typedef enum cores { PRETO = 0,VERDE, AZUL, CIANO, VERMELHO, AMARELO,ROXO, BRANCO } cores_t;
+typedef enum cores { PRETO = 0,VERDE, AZUL, CIANO, VERMELHO, AMARELO, ROXO, BRANCO} cores_t;
+
+// DebugLEDS
+#define DEBUGLED1 C_LED_DEBUG1_Pin
+#define DEBUGLED2 C_LED_DEBUG2_Pin
+#define DEBUGLED3 C_LED_DEBUG3_Pin
```

Comment:
Isso vem dos macros do main.h? Se sim, é arriscado fazer esse alias, porque algum lugar que incluir esse global_definitions pode não incluir o main.h, e se usar esses defines, resultará em erro de compilação. Não recomendo incluir o main.h nesse header aqui, então recomendo reavaliar se esses defines são necessários, e se não é viável utilizar `C_LED_DEBUGx_Pin` diretamente. Caso contrário, faça esse define diretamente no .h da tarefa debugleds.

Reply:
fiz diretamente no debugleds.h, achei melhor pois o texto é mais simples e não preciso incluir a main.h quando for usar os leds